obj/generators.h

```diff
@@ -46,8 +46,8 @@ namespace generators
     public:
         PointSource() = delete;
         explicit PointSource::PointSource(const double &freq, const size_t &numOfRaysPerRow, \
-                                        const double &diffusorSize, RandomGen* randomGen) : _frequency(freq),\
-                                        _numOfRaysPerRow(numOfRaysPerRow), _sampleSize(diffusorSize), \
+                                        const double &SampleSize, RandomGen* randomGen) : _frequency(freq),\
```

Comment:
Rename to sampleSize.

Be consistent with the naming, you use cammelCase with lower-cased first letter for variables.

Reply:
This was an accident. Will fix it